### PR TITLE
IdentityAndConsentManagement meeting notes 2023-07-12

### PR DESCRIPTION
**Pull request by jpengar**
#### What type of PR is this?
* documentation

#### What this PR does / why we need it:

MoM of the call held in 2023-07-12.

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: documentation/MeetingMinutes/MOM-2023-07-12.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+| [#32](https://github.com/camaraproject/IdentityAndConsentManagement/issues/32) | Orange | **The concept of purpose**<br> Sebastien (Orange) indicates that after analysis, they don’t see benefit in the use of purpose.<br>Discussion between Telefonica and Orange, regarding the implications. Telefonica states that scopes need not be changed if purposes are used.<br>Klaus (Vodafone) states that purpose may be different per country, so that’s a reason to have the concept separated<br>Proposal for way forward, as this issue is lasting long, is to have input on how to solve a practical scenario, using Orange proposal and using Telefonica proposal. In this way, the two solutions could be compared more easily and all participants could agree on the final solution (one of them or any other compromise solution in between that might be agreed).<br>Shilpa (DT) challenges whether there is a need to provide the purpose, if that is agreed in the onboarding phase. |
```
Comment (shilpa-padgaonkar) [inline documentation/MeetingMinutes/MOM-2023-07-12.md:84]: I would propose to change the line "Shilpa (DT) challenges whether there is a need to provide the purpose, if that is agreed in the onboarding phase. " with
Shilpa (DT) raised the question regarding how the final legal consent is derived in a case where multiple purposes are agreed with the ASP and the different purposes could indicate a different legal basis. And if just a single purpose is agreed with the ASP at the time of onboarding, why do we need to pass it within an API request. 
Jesus (TEF) mentioned that in such cases where multiple purposes are involved, some kind of policy will have to be agreed to ensure that the user experience is not impacted and that we can derive the needed legal basis (please feel free to change your content if I have not interpreted it correctly).

Comment (jpengar) [reply documentation/MeetingMinutes/MOM-2023-07-12.md]: Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/51/commits/c4964a20255e6aa763227ec5d25d8d0e5cef033c

**Review by shilpa-padgaonkar — APPROVED** [review]
/LGTM

**Comment by jpengar** [comment]
@rartych Could you give your approving review to merge the notes? Thank you very much!

**Pull request merged**